### PR TITLE
Support simultaneous deployments in different regions

### LINKED ISSUE #52: [FEATURE] Able to run multiple deployments on a client machine using different install dir
**Is your feature request related to a problem? Please describe.**
The script is not able to on multiple install directory in parallel. I am not able to do two deploys at a time.

**Describe the solution you'd like**
I should be able to do more than 1 deployment using different install/config directory.

**Describe alternatives you've considered**
The problem is the ibmcloud CLI config is placed in HOME directory of current user. We can change that using environment variable `IBMCLOUD_HOME=<install-dir>`.

**Additional context**
The ibmcloud CLI config stores the current target CRN in `~/.bluemix/plugins/power-iaas/config.json` making it diffult to run more than one deployment on different regions or service instances.

### PR DESCRIPTION
**Pull request by yussufsh**
Fixes #52

Signed-off-by: Yussuf Shaikh <yussuf.shaikh@ibm.com>

### CODE REVIEW

**Review by Prajyot-Parab — APPROVED** [review]
/lgtm

File: openshift-install-powervs [hunk 1]
```diff
@@ -85,6 +85,9 @@ BOOTSTRAP_DELETE_VARFILE="bootstrap-delete.tfvars"
 
 OLD_IFS=$IFS
 
+# All the ibmcloud configs and targeted power-iaas CRN will be stored under current directory named '.bluemix'
```
Comment (bpradipt) [inline openshift-install-powervs:88]: why bluemix ;) ?

**Comment by bpradipt** [comment]
Wondering why the macos check failed ? It doesn't show any logs

File: openshift-install-powervs (hunk 1 above)
Comment (yussufsh) [reply openshift-install-powervs:88]: It is as it from ibmcloud CLI.

**Comment by yussufsh** [comment]
> 
> 
> Wondering why the macos check failed ? It doesn't show any logs

GH Action giving issues recently on same host.

**Review by bpradipt — APPROVED** [review]
/lgtm

**Pull request merged**